content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx

```diff
@@ -0,0 +1,382 @@
+  title VARCHAR(255) NOT NULL,
+  createdAt TIMESTAMP NOT NULL DEFAULT now(),
+  content TEXT,
+  published BOOLEAN NOT NULL DEFAULT false,
+  authorId INTEGER NOT NULL,
+  FOREIGN KEY (authorId) REFERENCES User(id)
+);
+
+CREATE TABLE Profile (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  bio TEXT,
+  userId INTEGER UNIQUE NOT NULL,
+  FOREIGN KEY (userId) REFERENCES User(id)
+);
+```
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'sqlserver']}>
+
+```sql file=prisma/migrations/init/migration.sql
+CREATE TABLE [dbo].[Post] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [createdAt] DATETIME2 NOT NULL CONSTRAINT [Post_createdAt_df] DEFAULT CURRENT_TIMESTAMP,
+    [updatedAt] DATETIME2 NOT NULL,
+    [title] VARCHAR(255) NOT NULL,
+    [content] NVARCHAR(1000),
+    [published] BIT NOT NULL CONSTRAINT [Post_published_df] DEFAULT 0,
+    [authorId] INT NOT NULL,
+    CONSTRAINT [Post_pkey] PRIMARY KEY ([id])
+);
+
+CREATE TABLE [dbo].[Profile] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [bio] NVARCHAR(1000),
+    [userId] INT NOT NULL,
+    CONSTRAINT [Profile_pkey] PRIMARY KEY ([id]),
+    CONSTRAINT [Profile_userId_key] UNIQUE ([userId])
+);
+
+CREATE TABLE [dbo].[User] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [email] NVARCHAR(1000) NOT NULL,
+    [name] NVARCHAR(1000),
+    CONSTRAINT [User_pkey] PRIMARY KEY ([id]),
+    CONSTRAINT [User_email_key] UNIQUE ([email])
+);
+
+ALTER TABLE [dbo].[Post] ADD CONSTRAINT [Post_authorId_fkey] FOREIGN KEY ([authorId]) REFERENCES [dbo].[User]([id]) ON DELETE NO ACTION ON UPDATE CASCADE;
+
+ALTER TABLE [dbo].[Profile] ADD CONSTRAINT [Profile_userId_fkey] FOREIGN KEY ([userId]) REFERENCES [dbo].[User]([id]) ON DELETE NO ACTION ON UPDATE CASCADE;
+```
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'cockroachdb']}>
+
+```sql file=prisma/migrations/init/migration.sql
+CREATE TABLE "User" (
+  id INT8 PRIMARY KEY DEFAULT unique_rowid(),
+  name STRING(255),
+  email STRING(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE "Post" (
+  id INT8 PRIMARY KEY DEFAULT unique_rowid(),
+  title STRING(255) UNIQUE NOT NULL,
+  "createdAt" TIMESTAMP NOT NULL DEFAULT now(),
+  content STRING,
+  published BOOLEAN NOT NULL DEFAULT false,
+  "authorId" INT8 NOT NULL,
+  FOREIGN KEY ("authorId") REFERENCES "User"(id)
+);
+
+CREATE TABLE "Profile" (
+  id INT8 PRIMARY KEY DEFAULT unique_rowid(),
+  bio STRING,
+  "userId" INT8 UNIQUE NOT NULL,
+  FOREIGN KEY ("userId") REFERENCES "User"(id)
+);
+```
+
+</SwitchTech>
+
+Review the SQL migration file to ensure everything is correct.
+
+Next, mark the migration as applied using `prisma migrate resolve` with the `--applied` argument.
+
+```terminal
+npx prisma migrate resolve --applied init
+```
+
+The command will mark `init` as applied by adding it to the `_prisma_migrations` table.
+
+You can now make changes in your Prisma schema and use `prisma migrate dev` to evolve your database schema — generate and apply migrations to your database.
```

Comment:
Feel free to change the wording here, but would be nice to refer directly to the fact that you have now created baseline for your database

```suggestion
You now have a baseline for your current database schema. To make further changes to your database schema, you can update your Prisma schema and use `prisma migrate dev` to apply the changes to your database.
```